### ISSUE TITLE
Add only Admin UI to the Submissions Page

### ISSUE THREAD

**Issue by FlacoJones** [OpenQDev/OpenQ-Frontend#886]
Related to https://github.com/OpenQDev/OpenQ-Contracts/issues/80

**DESCRIPTION**

<img width="279" alt="Screenshot 2022-10-28 at 12 54 07 PM" src="https://user-images.githubusercontent.com/93455288/198701193-65a2f60f-4570-446f-b740-c71dc67ae76b.png">

I'm not sure how hard it would be with like D3.js or something to make the 3-tiered bar graph where each one is clickable? I just thought that was a cool UI component. Other ideas welcome.

Dependent on #883 

**WHATS NOT IN SCOPE**

Just make the UI - the contract call/OpenQClient updates can come second

No need to even make the tiers clickable
